packages/@eventual/aws-cdk/src/bucket-service.ts

```diff
@@ -141,6 +142,12 @@ export class BucketService<Service> {
         ],
       })
     );
+    if (this.props.compliancePolicy.isCustomerManagedKeys()) {
+      // data in the buckets are encrypted with a key that the customer owns
+      this.props.compliancePolicy.dataEncryptionKey.grantEncryptDecrypt(
+        grantee
+      );
+    }
```

Comment:
Not sure if I need to do this for AWS-managed keys. I don't have to for AWS-owned because there's zero visibility into that, but AWS-managed creates a key with an alias, like `aws/s3` and I'm hoping I don't need to grant permissions but I may.